composer.json

```diff
@@ -28,7 +28,7 @@
   ],
   "homepage": "https://github.com/auth0/auth0-PHP",
   "require": {
-    "php": "^8.0",
+    "php": "^8.1",
```

Comment:
PHP 8.0 has been deprecated, so we'll drop support with this coming release. This PR includes new language features that rely on 8.1+, so just going ahead and bumping it here.